test/e2e/storage/utils/utils.go

```diff
@@ -483,3 +487,43 @@ func CheckReadWriteToPath(pod *v1.Pod, volMode v1.PersistentVolumeMode, path str
 		VerifyExecInPodFail(pod, fmt.Sprintf("dd if=/dev/urandom of=%s bs=64 count=1", path), 1)
 	}
 }
+
+func genBinDataFromSeed(len int, seed int64) []byte {
```

Comment:
could we generate a string uuid instead? https://github.com/kubernetes/apimachinery/blob/ed135c5b96450fd24e5e981c708114fbbd950697/pkg/util/uuid/uuid.go#L30 it will be a bit easier to follow, don't need to worry about encoding then decoding it in the Write function

Reply:
nvm, there is no way to generate one from a seed